crates/vault-registry/src/types.rs

```diff
@@ -104,18 +107,15 @@ impl Wallet {
 }
 
 #[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, Debug)]
-pub enum VaultStatus {
+pub enum VaultStatus<SignedFixedPoint> {
     /// Vault is active - bool=true indicates that the vault accepts new issue requests
     Active(bool),
 
-    /// Vault has been liquidated
-    Liquidated,
-
-    /// Vault theft has been reported
-    CommittedTheft,
```

Comment:
Wasn't this kept to more easily determine what happened to the vault?

Reply:
I don't see any reason why would care once the vault has been liquidated. If this important for the front-end it is easily tracked externally. The reason I removed this is because I needed to define this redeem exchange rate and didn't want two separate instantiations.